src/lights/Light.js

```diff
@@ -50,6 +50,8 @@ Light.prototype = Object.assign( Object.create( Object3D.prototype ), {
 		if ( this.decay !== undefined ) data.object.decay = this.decay;
 		if ( this.penumbra !== undefined ) data.object.penumbra = this.penumbra;
 
+		if ( this.castShadow && this.shadow !== undefined ) data.object.shadow = this.shadow.toJSON();
```

Comment:
I wonder if the `this.castShadow` check is needed. Thinking about the editor case... One could set the shadow parameters but have the `castShadow` temporarily `false`. The shadow parameters would then get lost.